dev/breeze/src/airflow_breeze/params/shell_params.py

```diff
@@ -173,15 +174,18 @@ def print_badge_info(self):
             get_console().print(f'[info]Backend: {self.backend} {self.backend_version}[/]')
             get_console().print(f'[info]Airflow used at runtime: {self.use_airflow_version}[/]')
 
-    def get_backend_compose_files(self, backend: str):
+    def get_backend_compose_files(self, backend: str) -> list[str]:
         backend_docker_compose_file = f"{str(SCRIPTS_CI_DIR)}/docker-compose/backend-{backend}.yml"
         backend_port_docker_compose_file = f"{str(SCRIPTS_CI_DIR)}/docker-compose/backend-{backend}-port.yml"
-        return backend_docker_compose_file, backend_port_docker_compose_file
+        if backend == 'sqlite' or not self.forward_ports:
```

Comment:
Here (and few lines below) @bowrna I check if "sefl.forward_ports" is set to True - and only there I also include `-port.yml` files. Effectively all the `docker-compose` used by parallel tests do not forward the ports from host so the prblem with busy port is gone.

Reply:
That's why I told it was a bit `tricky` to pull off

Reply:
I see. How do you set the port to use in case of parallel tests then @potiuk ?

Reply:
Not needed any more :). Tests are run without Port forwarding.